script/cms_daily_report.rb

```diff
@@ -0,0 +1,271 @@
+# Total Enrolled Member count
+member_count = HbxEnrollment.collection.aggregate([
+  {"$match" => {
+      "hbx_enrollment_members" => {"$ne" => nil},
+      "external_enrollment" => {"$ne" => true},
+      "coverage_kind" => "health",
+      "consumer_role_id" => {"$ne" => nil},
+      "product_id" => { "$ne" => nil},
+      "aasm_state" => {"$in" =>  HbxEnrollment::RENEWAL_STATUSES + HbxEnrollment::ENROLLED_STATUSES},
+      "effective_on" => {"$gte" => Date.new(2022,1,1)}
+  }
+  },
+  {"$project" => {"member_count" => {"$size" => "$hbx_enrollment_members"}}},
+  {"$group" => {"_id" => 1, "total_members" => {"$sum" => "$member_count"}}}
+])
+​total_member_enrolled = member_count.first['total_members']
+
+# New Consumers Enrolled
+new_enrolled_members = 0
```

Comment:
Consumers who are actively enrolled in a health plan purchased after 11/1 AND were not auto-renewed